translations/messages.fr.xlf

```diff
@@ -0,0 +1,3070 @@
+      </trans-unit>
+      <trans-unit id="1207528295664437538" datatype="html">
+        <source>Stack your pieces and control as many stacks as you can to win!</source>
+        <target>Empilez vos pièces pour en contrôler un maximum et gagner !</target>
+      </trans-unit>
+      <trans-unit id="7930050431770016664" datatype="html">
+        <source>An enhanced tic-tac-toe where piece can encapsule other and prevent them to win.</source>
+        <target>Un moripion amélioré où les pièces peuvent en encapsuler d'autres pour éviter la défaite.</target>
+      </trans-unit>
+      <trans-unit id="8971165322320863634" datatype="html">
+        <source>An antiquity-war inspired game. Be the first to pierce your opponent&apos;s lines!</source>
+        <target>Un jeu inspiré de l'antiquité. Soyez le premier à percer les lignes ennemies !</target>
+      </trans-unit>
+      <trans-unit id="6403042813790072176" datatype="html">
+        <source>A hexagonal game of alignment. Don&apos;t be the first one to be out of pieces to place on the board!</source>
+        <target>Un jeu hexagonal d'alignement. Ne soyez pas le premier à être à court de pièces à placer !</target>
+      </trans-unit>
+      <trans-unit id="6676975125770922470" datatype="html">
+        <source>The oldest strategy game still practiced widely. A territory control game.</source>
+        <target>Le plus vieux jeu de stratégie encore joué. Un jeu de contrôle de territoire</target>
+      </trans-unit>
+      <trans-unit id="8165475229121998889" datatype="html">
+        <source>Regroup your pieces to win. But your possible moves will often change!</source>
+        <target>Regroupez vos pièces pour gagner. Mais les mouvements possibles changent constamment !</target>
+      </trans-unit>
+      <trans-unit id="287142221400627248" datatype="html">
+        <source>The classical 4 in a row game!</source>
+        <target>Le classique Puissance 4 !</target>
+      </trans-unit>
+      <trans-unit id="6377255794894182998" datatype="html">
+        <source>Drop a piece, then rotate the board. The first player to align 5 pieces wins!</source>
+        <target>Posez une pièces, ensuite tournez un quadrant. Le premier à aligner 5 pièces gagne !</target>
+      </trans-unit>
+      <trans-unit id="3798499751517521657" datatype="html">
+        <source>Overlay your pieces and use two game mechanics to conserve your pieces. First player to run out of pieces loose!</source>
+        <target>Superposez vos pièces et utilisez deux mécaniques de jeux pour conserver vos pièces. Le premier joueur qui n'a plus de pièce perd !</target>
+      </trans-unit>
+      <trans-unit id="1827371853303540301" datatype="html">
+        <source>Immobilize one of your opponent&apos;s pyramids before your opponent does!</source>
+        <target>Immobilisez une des pièces de votre adversaire avant qu'il ne le fasse !</target>
+      </trans-unit>
```

Comment:
Soyez le premier à immobiliser une pièce de l'adversaire ?

Reply:
Et en anglais on peut dire la même chose ou ne point changer, foerte :D 🗡️

Reply:
Pas nécessaire de traduire litéralement, donc why not

Reply:
Cependant, on pourrait quand même garder "pyramide" ?